models.json

```diff
@@ -381,5 +381,69 @@
         }
       }
     }
+  },
+  "Middleware": {
+    "properties": {
+      "id": {"type": "string", "id": true, "json": false},
+      "name": {"type": "string", "json": false},
+      "phase": {"type": "string", "json": false},
+      "subPhase": {"type": "string", "json": false},
+      "isPhasePlaceHolder": {"type": "boolean", "json": false},
+      "nextPhase": {"type": "string", "json": false},
+      "order": {"type": "number", "json": false},
+      "paths": {"type": ["string"]},
+      "params": {"type": "object"},
+      "facetName": {"type": "string", "required": true, "json": false}
+    },
+    "public": true,
+    "dataSource": "db",
+    "options": {
+      "base": "Definition",
+      "defaultConfigFile": "middleware.json",
+      "configFiles": [
+        "middleware.json",
+        "middleware.*.json"
+      ],
+      "relations": {
+        "facet": {
+          "type": "belongsTo",
+          "model": "Facet",
+          "foreignKey": "facetName"
+        }
+      }
+    }
+  },
+  "MiddlewarePhase": {
```

Comment:
It looks like `MiddlewarePhase` is defined in JSON, but it is never created nor referenced anywhere. It looks like a dead code to me. We should either remove this model, or else put it to use. Thoughts?